pkg/antctl/command_definition.go

```diff
@@ -78,7 +78,7 @@ const (
 	query
 )
 
-var groupCommands = map[commandGroup]*cobra.Command{
+var GroupCommands = map[commandGroup]*cobra.Command{
```

Comment:
why is it made public?

Reply:
the featuregates is in raw command, it doesn't use the command definition framework which has the "get" sub command. since we want to put featuregates to "get" sub command, we need to make it public so raw command can also add itself as "get" sub command. otherwise, raw command will always be the command after `antctl`

Reply:
It doesn't seem that you need to make it public for this purpose. All the methods that access the map are in this package and I don't think you need to access the map directly from other packages to insert a command. Doesn't it use `applyToRootCommand` to do this?

Reply:
yes, it uses applyToRootCommand, @jianjuns 's suggestion is it will be better to put it under "get" sub command, so  I refine the code to make sure if there is a sub command group provided in raw command def, we can add it.

Reply:
My point is "It doesn't seem that you need to make it public for this purpose."

Reply:
ah, you are right, they are in the same package, I probably make it public for other method at very beginning. let me update it. thanks.